recipes-kernel/linux/linux-hikey-aosp_4.4.bb

```diff
@@ -71,4 +71,6 @@ do_configure() {
     bbplain "Saving defconfig to:\n${B}/defconfig"
     oe_runmake -C ${B} savedefconfig
     cp -a ${B}/defconfig ${DEPLOY_DIR_IMAGE}
+    bbplain "Saving .config to:\n${B}/config"
+    cp -a ${B}/.config ${DEPLOY_DIR_IMAGE}/config
```

Comment:
@koenkooi the goal is to present it as URL in the snapshots (example: https://qa-reports.linaro.org/lkft/linux-mainline/build/v4.13-rc1-72-gbeaec53/). Defconfig is already there.